examples/queue/index.ts

```diff
@@ -6,27 +6,68 @@ const resourceGroup = new azure.core.ResourceGroup("resourcegroup", {
     location: "West US 2",
 });
 
-// Create a storage account for our images
+// Create a storage account for our queues
 const storageAccount = new azure.storage.Account("storage", {
     resourceGroupName: resourceGroup.name,
     accountReplicationType: "LRS",
     accountTier: "Standard",
 });
 
-// And a queue to send messages into
-const queue = new azure.storage.Queue("queue", {
+// And an input queue to send messages into
+const queue1 = new azure.storage.Queue("queue1", {
    resourceGroupName: resourceGroup.name,
-   storageAccountName: storageAccount.name
+   storageAccountName: storageAccount.name,
 });
 
-// When a new message is added, fire an event
-queue.onEvent("newMessage",  async (context, arg) => {
-    console.log("ctx: " + JSON.stringify(context, null, 4));
-    console.log("arg: " + JSON.stringify(arg, null, 4));
+// And an input queue to send messages into
+const queue2 = new azure.storage.Queue("queue2", {
+    resourceGroupName: resourceGroup.name,
+    storageAccountName: storageAccount.name,
+ });
+
+// HTTP Function will send a message to the first queue on each request
+const greeting = new azure.appservice.HttpEventSubscription('greeting', {
+    resourceGroup,
+    route: "{name}",
+    inputOutputs: [
```

Comment:
why `inputOutputs` and not `bindings`? Or 2 distinct properties, 1 for inputs, 1 for outputs?

Reply:
I had both of your versions in the past drafts :) 

I thought `bindings` *might* be confusing because they don't include the trigger (and the HTTP output), while technically a trigger is also a binding.

Two separate properties could make sense... I guess I'd need to have two distinct types then, to avoid misplacing?

Reply:
Your remark about `bindings` *might* be true. I personally would prefer to have 2 separate properties, but that’s not a strong opinion. If you choose to make it into two properties, you could probably relatively easy prevent misplacing by adding `& {direction:“out“}` to the type definition of outputs (and similar for inputs)

Reply:
i think i woudl prefer two separate properties as well.

Reply:
Done, I like it too